sql/moz-fx-data-shared-prod/backfills_staging_derived/baseline_clients_last_seen_v1_20211130/metadata.yaml

```
@@ -0,0 +1,32 @@
+---
+# Generated via bigquery_etl.glean_usage
+
+friendly_name: Baseline Clients Last Seen
+description: |-
+  Captures activity history of each client in 28-day
+  windows for each submission date based on baseline pings.
+
+  Clustering fields: `normalized_channel`, `sample_id`
+
+  See also: `baseline_clients_daily`
+owners:
+  - ascholtz@mozilla.com
+labels:
+  incremental: true
+  schedule: daily
+scheduling:
+  depends_on_past: true
+bigquery:
+  time_partitioning:
+    type: day
+    field: submission_date
+    require_partition_filter: true
+  clustering:
+    fields:
+      - normalized_channel
+      - sample_id
+monitoring:
+  enabled: true
+schema:
+  derived_from:
+    - table: [ 'moz-fx-data-shared-prod', 'firefox_desktop_derived', 'baseline_clients_daily_v1' ]
```

This is causing the `verify-format-yaml` CI check to fail with
```
./sql/moz-fx-data-shared-prod/backfills_staging_derived/baseline_clients_last_seen_v1_20211130/metadata.yaml
  32:15     error    too many spaces inside brackets  (brackets)
  32:97     error    too many spaces inside brackets  (brackets)
```
```suggestion
    - table: ['moz-fx-data-shared-prod', 'firefox_desktop_derived', 'baseline_clients_daily_v1']
```